src/main/infrastructure/repository/http/HttpClient.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * @interface
+ */
+class HttpClient {
+  post({url, data, options}) {}
```

Comment:
Why do we need in the interface the post? If the AxiosHttpClient doesn't implement it?

Reply:
well seen, that's a copy-paste bug xD will change